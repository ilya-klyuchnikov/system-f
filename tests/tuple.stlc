(let ((t (tup 3 4 5 6)))
  (tup (prj t 2) 6 7))
