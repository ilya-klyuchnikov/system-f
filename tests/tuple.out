>  : (* int int int)
-> (tup 5 6 7)
> 